examples/gallery/plot/multi_parameters.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
Please rename the file to a better name, maybe "multi-parameter-symbols"?

Reply:
done!